### PR TITLE
Deploy docs

### PR DESCRIPTION
**Pull request by tarunipaleru**

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -106,4 +106,41 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
...
+```
+
+Customize the `name` and `branches` variables to your specifications. In most cases branches should be set to your main production branch. 
+
+
+> If you're using Personal Access Tokens to push up your code to Github, make sure in *Profile Settings --> Developer Access --> Personal access tokens* that `workflow` access is enabled for your token. 
+
+In your Hathora project folder in CLI, run the code below to generate and copy your secret to clipboard:
+```
+cat ~/.config/hathora/token | pbcopy
```
Comment (hpx7) [inline docs/deploy.md]: is `pbcopy` available on windows?

File: docs/deploy.md [hunk 2]
```diff
@@ -106,4 +106,41 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
...
+jobs:
+  server:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install -g hathora
+      - run: hathora cloud deploy --appName XXXXXXXXX --token ${{ secrets.HATHORA_TOKEN }}
+```
+
+Customize the `name` and `branches` variables to your specifications. In most cases branches should be set to your main production branch.
```
Comment (hpx7) [inline docs/deploy.md]: they should also customize the `appName`

File: docs/deploy.md [hunk 3]
```diff
@@ -106,4 +106,60 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
 10. To destory your application run `hathora cloud destroy --appName yourAppName`
 
 
-Self-hosting will remain an option for those that want more control.
\ No newline at end of file
+Self-hosting will remain an option for those that want more control.
+
+## Setting up Continuous Delivery with Github Actions
+
+Here you will learn how to set up continuous delivery with Github Actions but you can use the platform of your choice.
```
Comment (hpx7) [inline docs/deploy.md]: maybe rephrase this sentence

File: docs/deploy.md [hunk 4]
```diff
@@ -106,4 +106,60 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
...
 
-Self-hosting will remain an option for those that want more control.
\ No newline at end of file
+Self-hosting will remain an option for those that want more control.
+
+## Setting up Continuous Delivery with Github Actions
+
+Here you will learn how to set up continuous delivery with Github Actions but you can use the platform of your choice. 
+
+In your Github Repo, select the actions tab and create a custom `.yml` file. Remove the auto-generated code and replace with:
```
Comment (hpx7) [inline docs/deploy.md]: you can include something like:

"you can name the file whatever you want, for example `deploy.yml`

**Review by dsiddharth — APPROVED** [review]

File: docs/deploy.md [hunk 5]
```diff
@@ -106,4 +106,60 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
...
+          --auth '${{ secrets.NETLIFY_AUTH_TOKEN }}' \
+          --dir=client/web/dist;
+```
+
+Customize the `name`, `branches`, and `appName` variables to your specifications. In most cases branches should be set to your main production branch. 
+
+
+> If you're using Personal Access Tokens to push up your code to Github, make sure in *Profile Settings --> Developer Access --> Personal access tokens* that `workflow` access is enabled for your token. 
+
+In your Hathora project folder in CLI, run the command below to generate your Hathora secret to clipboard.
```
Comment (hpx7) [inline docs/deploy.md]: instead of specifying the command, just say something like

"copy the token from `~/.config/hathora/token`"

Comment (hpx7) [reply docs/deploy.md]: and keep the warning
